### PR TITLE
pic crate cleanups

### PR DESCRIPTION
**Pull request by 0xAX**
Hello @steveklabnik,

Here are two simple commits:

1. Provides some constants to use it in the pic crate. From my view it does code more readable and delivers us from magic numbers in some places.

2. Documentation for `remap()`

### CODE REVIEW

File: pic/src/lib.rs [hunk 1]
```diff
@@ -4,33 +4,56 @@
 extern crate x86;
 use x86::shared::io::{inb, outb};
 
+/* PIC input/output ports */
```
Comment (steveklabnik) [inline pic/src/lib.rs]: Could you use `//` instead of `/*` please?

File: pic/src/lib.rs [hunk 2]
```diff
@@ -4,33 +4,56 @@
...
+const PIC1_CMD_IO_PORT: u16 = 0x0020;
+const PIC2_CMD_IO_PORT: u16 = 0x00A0;
+const PIC1_DATA_IO_PORT: u16 = 0x0021;
+const PIC2_DATA_IO_PORT: u16 = 0x00A1;
+
+/* PIC commands */
```
Comment (steveklabnik) [inline pic/src/lib.rs]: same here

File: pic/src/lib.rs [hunk 3]
```diff
@@ -4,33 +4,56 @@
...
+const ICW1: u8 = 0x11;
+const ICW4: u8 = 0x1;
+
+/* new inetrrupt vector offsets for remmaped PICs */
```
Comment (steveklabnik) [inline pic/src/lib.rs]: and here

File: pic/src/lib.rs [hunk 4]
```diff
@@ -4,33 +4,56 @@
 extern crate x86;
 use x86::shared::io::{inb, outb};
 
+/* PIC input/output ports */
+const PIC1_CMD_IO_PORT: u16 = 0x0020;
```
Comment (steveklabnik) [inline pic/src/lib.rs]: is there a good reason to use `0x0020` instead of `0x20`?

**Comment by 0xAX** [comment]
@steveklabnik everything is fixed. Thanks for feedback.

File: pic/src/lib.rs (hunk 4 above)
Comment (0xAX) [inline pic/src/lib.rs]: The `x86::shared::io.outb()` takes `u16` as second parameter

Comment (steveklabnik) [inline pic/src/lib.rs]: Ah, thanks.

**Comment by steveklabnik** [comment]
Thanks!

**Pull request merged**